lib/api/src/modules/stories.ts

```diff
@@ -496,6 +500,8 @@ export const init: ModuleFn = ({
       }
     );
 
+    indexClient = new StoryIndexClient();
+    indexClient.addEventListener('INVALIDATE', () => fullAPI.fetchStoryList());
```

Comment:
can you explain the flow of control here?

Reply:
The server code detects a change, triggers the invalidation event. The client code picks up the event, triggering the fetch stories index code to refetch it.